backends/es6/publication_index.go

```diff
@@ -310,16 +310,16 @@ func (pi *PublicationIndex) DeleteAll() error {
 	}
 	res, err := req.Do(ctx, pi.client)
 	if err != nil {
-		return err
+		return fmt.Errorf("publicationindex.DeleteAll: es6 http error: %w", err)
 	}
 	defer res.Body.Close()
 
 	if res.IsError() {
 		buf := &bytes.Buffer{}
 		if _, err := io.Copy(buf, res.Body); err != nil {
-			return err
+			return fmt.Errorf("publicationindex.DeleteAll: io error while reading es6 error response body: %w", err)
 		}
-		return errors.New("Es6 error response: " + buf.String())
+		return errors.New("publicationindex.DeleteAll: es6 error response: " + buf.String())
```

Comment:
Out of curiosity: why uses `errors.New` here and not `ftm.Errorf`?

Reply:
If it is just to concatenate strings, I do not see any gain in using `fmt.Errorf`. 

`Errors.New("my " + "concatenated string")` 

vs

`fmt.Errorf("my %s", "concatenated string")`

If you trying to insert anything other than strings (float, integer, and even a wrapped error), that is another case. `Errors.New("my length: " + fmt.Sprintf("%d", 186))` would make it too verbose, so `fmt.Errorf("my length: %d", 186)` is more concise.